### PR TITLE
Edit travis.yml, Travis always builds on node 4

### LINKED ISSUE #154: Compiler warning about babel-preset-es2015@6.24.1
happens during `npm install`

```
npm WARN deprecated babel-preset-es2015@6.24.1: We're super 😸  excited that you're trying to
use ES2015 syntax, but instead of continuing yearly presets 😭 , we recommend using 
babel-preset-env: npm install babel-preset-env. preset-env without options will compile ES2015+
down to ES5. And by targeting specific browsers, Babel can do less work and you can ship native 
ES2015+ to users 😎 ! Also, we are in the process of releasing v7, so give 
http://babeljs.io/blog/2017/09/12/planning-for-7.0 a read and test it! Thanks so much for using 
Babel 🙏 , please give us a follow @babeljs for updates, join slack.babeljs.io for discussion and help 
support at opencollective.com/babel
```

### PR DESCRIPTION
**Pull request by MarckK**
Use babel-preset-env instead of babel-preset-2015
Resolves #154
Signed-off-by: Kara de la Marck <karadelamarck@gmail.com>

### CODE REVIEW

**Comment by MarckK** [comment]
This PR follows the configuration in [opentracing-javascript travis.yml file](https://github.com/opentracing/opentracing-javascript/blob/master/.travis.yml).

File: .travis.yml [hunk 1]
```diff
@@ -44,9 +42,32 @@ before_install:
 - npm install babel-cli
 - if [ "$CROSSDOCK" = "1" ]; then make install_docker_ci ; fi
 
+before_script:
+# Install the Node version to test against
+- nvm install $TEST_NODE_VERSION
+- nvm use $TRAVIS_NODE_VERSION
```
[the remark this answers is not in the document]
Comment (MarckK) [reply .travis.yml]: Thank you, Yuri for your review! :)

File: .travis.yml [hunk 2]
```diff
@@ -44,9 +42,32 @@ before_install:
...
+
 script:
-- if [ "$CROSSDOCK" != "1" ]; then make test ; fi
-- if [ "$COVER" = "1" ]; then npm run coveralls ; fi
+- if [ "$CROSSDOCK" != "1" ]; then
```
[the remark this answers is not in the document]
Comment (MarckK) [reply .travis.yml]: 👍

**Comment by yurishkuro** [comment]
I've restarted the build several time but it keeps failing, seemingly due to Cassandra taking too long. I see that in the main repo the test has a much longer timeout, could you make a similar change here? https://github.com/jaegertracing/jaeger/blob/master/crossdock/docker-compose.yml#L18

**Comment by yurishkuro** [comment]
Fantastic

**Pull request merged**